java/runtime/src/test/java/io/ray/runtime/functionmanager/FunctionManagerTest.java

```diff
@@ -27,11 +27,24 @@ public static Object foo() {
     return null;
   }
 
-  public static class Bar {
+  public static class Base {
+
+    public Object foo() {
+      return null;
+    }
+
+    public Object bar() {
+      return null;
+    }
+
+  }
+
+  public static class Bar extends Base {
```

Comment:
```suggestion
  public static class ChildClass extends ParentClass {
```

The class name "Bar" and method name "bar" are confusing to me.